src/sentry/models/deletedproject.py

```diff
@@ -22,6 +22,7 @@ class DeletedProject(DeletedEntry):
     organization_name = models.CharField(max_length=64, null=True)
     organization_slug = models.CharField(max_length=50, null=True)
 
+    # TODO(jess): ask about how to deal with these
     team_id = BoundedBigIntegerField(null=True)
```

Comment:
not sure if these should just be removed and project/org is detailed enough?

Reply:
Yeah, I think it'd be fair to drop this.

Reply:
ok, will stop writing to it in this pr (https://github.com/getsentry/sentry/pull/6807/commits/a3e514511b00a46e5e0c02c826bb1e5fcf18cdb5) and put up https://github.com/getsentry/sentry/pull/6856 to remove it completely